src/pages/preferences/modals/ResetPasswordModal.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <va-modal 
+    hideDefaultActions
+    modelValue
+    :mobileFullscreen="false"
+    @update:modelValue="emits('cancel')"
+  >
+    <va-form 
+      class="space-y-6 w-[326px] md:w-[608px]"
+      ref="form"
+      @submit.prevent="submit"
+    >
+      <div class="grid grid-cols-1 md:grid-cols-2 gap-6">
+        <va-input
+          requiredMark
+          v-model="oldPassowrd"
+          label="Old password"
+          placeholder="Old password"
+          type="password"
+          :rules="oldPasswordRules"
+        />
+        <div class="hidden md:block" />
+        <va-input
+          requiredMark
```

Comment:
* Check marks should be operational. Icon should be cross on invalid rule, check on valid.
* For password field - mention in messages that current password is 11111111. Right now 12345678 is hard to guess.

![image](https://github.com/epicmaxco/vuestic-admin/assets/5394573/8f8e21c8-536a-4f3f-81c1-fe16019051dd)